documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,72 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo \
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo dnf config-manager --add-repo \
+	https://yum.dockerproject.org/repo/main/fedora/24
```

Comment:
I think we need a clear comment explaining that, yes, we really do mean Fedora 24 here and everywhere else in the doc.

Without that, it's not unreasonable for the user to think that we only partially updated this doc from F24->F25 and the user may then disregard all mention of F24, go with F25 packages and have a bad outcome.